sql-parser/src/main/java/io/crate/sql/parser/Statement.g

```diff
@@ -1280,6 +1300,7 @@ KEY: 'KEY';
 DUPLICATE: 'DUPLICATE';
 SET: 'SET';
 RESET: 'RESET';
+DEFAULT: 'DEFAULT';
```

Comment:
is it possible to add default to nonReserved as well? I think otherwise this is a breaking change.
`create table t (default string)` will fail and default needs to be quoted: `create table t ("default" string)`